File: C/3190FindMinimumOperationsToMakeAllElementsDivisibleByThree.c
#include <stdio.h>

int minimumOperations(int* nums, int numsSize){
    int counter = 0;

    for(int i = 0; i < numsSize; i++){
        if(nums[i] % 3 != 0){
            int k = nums[i] % 3;
            if(3 - k > k){
                counter += k;
            }else{
                counter += (3 - k);
            }
        }
    }

    return counter;
}

int main(){

    int nums[] = {1, 2, 3, 4};
    int numsSize = 4;
    printf("%d", minimumOperations(nums, numsSize));

    return 0;
}

File: Python/3190FindMinimumOperationsToMakeAllElementsDivisibleByThree.py
def minimumOperations(nums : list[int]) -> int:
    counter = 0

    for num in nums:
        if num % 3 != 0:
            k = num % 3
            if 3 - k > k:
                counter += k
            else:
                counter += (3 - k)
    
    return counter

nums = [1, 2, 3, 4]
print(minimumOperations(nums))